app/models/shipit/user.rb

```diff
@@ -123,5 +123,28 @@ def identify_renamed_user!
     rescue Octokit::NotFound
       false
     end
+
+    def org_email?(email_address)
+      return false if email_address.blank?
+
+      org_domains = Shipit.preferred_org_emails
+      return true if org_domains.blank?
+
+      org_domains.any? { |domain| email_address.end_with?("@#{domain}") }
+    end
+
+    def appropriate_email_for(github_user)
+      return github_user.email if org_email?(github_user.email)
+
+      begin
+        github_api.emails
```

Comment:
I figure I should mention,
I am more of a fan of this approach:
```ruby
github_api
  .emails
  .sort_by { |e| e.primary ? 0 : 1 }
  .map(&:email)
  .find { |e| org_email?(e) }
```
but it violates our rubocop settings.

I looked through various projects using a regex to find method chaining styles, and this ones was not uncommon.